diff -up irqbalance-1.0.5/misc/irqbalance.service.orig irqbalance-1.0.5/misc/irqbalance.service
--- irqbalance-1.0.5/misc/irqbalance.service.orig	2013-06-19 17:02:29.426876009 +0200
+++ irqbalance-1.0.5/misc/irqbalance.service	2013-06-19 17:02:40.881876787 +0200
@@ -1,6 +1,7 @@
 [Unit]
 Description=irqbalance daemon
 After=syslog.target
+ConditionVirtualization=no
 
 [Service]
 EnvironmentFile=/etc/sysconfig/irqbalance
