diff -up irqbalance-1.0.6/misc/irqbalance.service.orig irqbalance-1.0.6/misc/irqbalance.service
--- irqbalance-1.0.6/misc/irqbalance.service.orig	2013-06-19 16:51:49.586832569 +0200
+++ irqbalance-1.0.6/misc/irqbalance.service	2013-06-19 16:52:02.385833438 +0200
@@ -1,6 +1,7 @@
 [Unit]
 Description=irqbalance daemon
 After=syslog.target
+ConditionVirtualization=no
 
 [Service]
 EnvironmentFile=/etc/sysconfig/irqbalance
